x-pack/docs/en/security/authorization/mapping-roles.asciidoc

```diff
@@ -66,6 +66,16 @@ You can change this default behavior by changing the
 this is a common setting in Elasticsearch, changing its value might effect other
 schedules in the system.
 
+While the _role mapping APIs_ is he preferred way to manage role mappings, using
+the `role_mappings.yml` file becomes useful if you want o define fixed role mappings
+that no one (besides an administrator with physical access to the {es} nodes) would
+be able to change. Please note however, that the role_mappings.yml file is provided
+as a minimal administrative function and is not intended to cover and be used to
+define roles for all use cases.
```

Comment:
I don't have a suggested wording, but the other use case is to allow authentication to work in a red cluster, similar to file-based users. (It's less commonly used, but if you want to have an LDAP/PKI managed superuser, then you probably want it to be a file based role mapping)

Reply:
I think the wording here merely tries to emphasize that API should be the way to go unless you _know_ why to use file based roles/role mappings. As such, I don't think we need to explicitly state all the use cases as this would make the paragraph large enough and the message might get lost. WDYT ?

Reply:
In that case, I would prefer to remove the use case that is mentioned.

If we want to give guidance about when it's a good idea to use the file, then ought to cover a range of use cases. Otherwise, listing one makes it _feel_ like guidance without actually being complete.

Reply:
ok, took a swing at adding the second use case too

Reply:
ping @tvernum for a review